lib/chef/resource/kernel_module.rb

```diff
@@ -16,11 +16,61 @@ class KernelModule < Chef::Resource
 
       description "Use the kernel_module resource to manage kernel modules on Linux systems. This resource can load, unload, blacklist, install, and uninstall modules."
       introduced "14.3"
+      examples <<~DOC
+        Install and load a kernel module, and ensure it loads on reboot.
+        ```ruby
+        kernel_module 'loop'
+        ```
+        Install and load a kernel with a specific set of options, and ensure it loads on reboot. Consult kernel module
+        documentation for specific options that are supported.
+        ```ruby
+        kernel_module 'loop' do
+          options [
+            'max_loop=4',
+            'max_part=8'
+          ]
+        end
+        ```
+        Load a kernel module.
+        ```ruby
+        kernel_module 'loop' do
+          action :load
+        end
+        ```
+        Unload a kernel module and remove module config, so it doesn’t load on reboot.
+        ```ruby
+        kernel_module 'loop' do
+          action :uninstall
+        end
+        ```
+        Unload kernel module.
+        ```ruby
+        kernel_module 'loop' do
+          action :unload
+        end
+        ```
+        Blacklist a module from loading.
+        ```ruby
+        kernel_module 'loop' do
+          action :blacklist
+        end
+        ```
+        Disable a kernel module.
+        ```ruby
+        kernel_module 'loop' do
+          action :disable
+        end
+        ```
+      DOC
 
       property :modname, String,
                description: "An optional property to set the kernel module name if it differs from the resource block's name.",
                name_property: true, identity: true
 
+      property :options, Array,
+        description: "An optional property to set options for the kernel module.",
+        introduced: "15.4"
```

Comment:
You'll need to update this for the next 14 release number

Reply:
Would that be ``14.15`` or something else?

Reply:
14.15 seems right